mix.exs

```diff
@@ -37,7 +42,9 @@ defmodule Feedback.Mixfile do
      {:phoenix_html, "~> 2.6"},
      {:phoenix_live_reload, "~> 1.0", only: :dev},
      {:gettext, "~> 0.11"},
-     {:cowboy, "~> 1.0"}]
+     {:cowboy, "~> 1.0"},
+     {:comeonin, "~> 2.0"},
```

Comment:
we only need `comeonin` if we are actually letting people save a password, which isn't the case yet, right? 🤔 
From what I can tell, this PR does not include a _form_ to actually allow a user to register.
also this will be the 5th time we have built user registration/login from scratch instead of abstracting it into a re-useable module ... 😞